proxy/http/HttpConnectionCount.h

```diff
@@ -21,237 +21,382 @@
   limitations under the License.
  */
 
-//
-#include "ts/ink_platform.h"
-#include "ts/ink_inet.h"
-#include "ts/ink_mutex.h"
-#include "ts/Map.h"
-#include "ts/Diags.h"
-#include "ts/CryptoHash.h"
-#include "ts/ink_config.h"
+#pragma once
+
+#include <string_view>
+#include <chrono>
+#include <atomic>
+#include <sstream>
+#include <tuple>
+#include <mutex>
+#include <ts/ink_platform.h>
+#include <ts/ink_config.h>
+#include <ts/ink_mutex.h>
+#include <ts/ink_inet.h>
+#include <ts/Map.h>
+#include <ts/Diags.h>
+#include <ts/CryptoHash.h>
+#include <ts/BufferWriterForward.h>
+#include <MgmtDefs.h>
 #include "HttpProxyAPIEnums.h"
 #include "Show.h"
-#include <sstream>
 
-#pragma once
+constexpr std::string_view operator"" _sv(const char *s, size_t n)
```

Comment:
This is annoying - for some reason using this literal constructor enables a `std::array<std::string_view>` to be `constexpr` when using `sv` doesn't. That seems seriously broken to me, but there it is.